src/v2/coins/algo.ts

```diff
@@ -0,0 +1,37 @@
+import BaseCoin = require('../baseCoin');
+import * as _ from 'lodash';
+
+
+class Algorand extends BaseCoin {
```

Comment:
Can we use `Algo` as class name for consistency?

Reply:
sure